lib/utils/index.js

```diff
@@ -340,6 +340,30 @@ export function nativeToJSError(code: string, message: string, additionalProps?:
   return error;
 }
 
+/**
```

Comment:
This has moved to lib/utils/native.js so doesn't need to be here anymore - I assume this was from a merge?